tests/unit/apps/account/test_user_import_csv.py

```diff
@@ -31,7 +31,7 @@ def setUpTestData(cls) -> None:
 
     @responses.activate
     def test_import_csv(self) -> None:
-        url = reverse("admin:account_user_import_csv")
+        url = reverse("admin:account_user_import_csv", args=[self.superuser.pk])
```

Comment:
these changes are related to not need required changes above

Reply:
I noticed this bug and fixed it